lib/i18n/tasks/command/commands/missing.rb

```diff
@@ -46,6 +47,20 @@ def translate_missing(opt = {})
           print_forest translated, opt
         end
 
+        # DeepL Pro Translate
+        cmd :translate_missing_deepl,
```

Comment:
Let's add a `--provider` flag to `translate_missing` instead of having a separate command that is otherwise identical.

Reply:
Could you check if `Command::Options::Locales` could be good as well?